src/main.js

```diff
@@ -94,12 +96,14 @@ class Entry {
    *@param {string|null} envType type of variable e.g. url
    *@param {Array|null} envChoices given restricted choices e.g. [data,info,1]
    *@param {string|null} envDefaultVal given default value e.g. data
+   *@param {string} envComment given comment using the # symbol
    */
-  constructor(envName, envType, envChoices, envDefaultVal) {
+  constructor(envName, envType, envChoices, envDefaultVal, envComment) {
     this.name = envName;
     this.type = envType;
     this.choices = envChoices;
     this.defaultValue = envDefaultVal;
+    this.comment = envComment;
```

Comment:
I suggest [`trim`ing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) the comment. There is no reason to maintain leading and trailing white spaces.